rust-runtime/aws-smithy-http-server/src/protocols.rs

```diff
@@ -166,7 +165,7 @@ mod tests {
 
     #[test]
     fn check_invalid_content_type() {
-        let invalid = vec!["application/ajson", "text/xml"];
+        let invalid = vec!["application/jason", "text/xml"];
```

Comment:
IntelliJ didn't like the spelling of "ajson" and I think "jason" is funnier anyways